packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -575,7 +575,20 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
      * @param fullTree - true to bypass optimizations and force a full summary tree
      * @param trackState - This tells whether we should track state from this summary.
      */
-    public async summarize(fullTree: boolean = false, trackState: boolean = true): Promise<ISummaryTreeWithStats> {
+    public async summarize(params: boolean | undefined | {
+        noHandles: boolean;
```

Comment:
this method signature is super wonky. it should be clean up, and documented. i think it's fine it we break backcompat